lib/internal/Magento/Framework/DataObject.php

```diff
@@ -387,22 +387,38 @@ public function toString($format = '')
      */
     public function __call($method, $args)
     {
-        switch (substr((string)$method, 0, 3)) {
+        switch ($method[0].($method[1] ?? '').($method[2] ?? '')) {
```

Comment:
Can we put this `$method[0].($method[1] ?? '').($method[2] ?? '')` into a variable? Not sure if this will have an extra performance impact but it should at least make the code a bit more readable 😄

Reply:
I added a commend to make it more obvious and not affect performance improvement